### PR TITLE
minor monitor-components improvements

### PR DESCRIPTION
**Pull request by rimrul**

### CODE REVIEW

File: .github/workflows/monitor-components.yml [hunk 1]
```diff
@@ -12,12 +12,11 @@ on:
 
 env:
   CHARACTER_LIMIT: 5000
-  MAX_AGE: 48h
+  MAX_AGE: 7d
 
 jobs:
   job:
     runs-on: ubuntu-latest
-    environment: monitor-components
```
Comment (dscho) [inline .github/workflows/monitor-components.yml:20]: I believe I added this to allow the workflow to open issues, using an environment secret.

A better way (that I was unaware of, at the time I created that environment) would probably be something along these lines:

```yaml
permissions:
  issues: write
```

**Comment by S3od11** [comment]
'

**Review by S3od11 — COMMENTED** [review]
'

File: .github/workflows/monitor-components.yml (hunk 1 above)
Comment (S3od11) [reply .github/workflows/monitor-components.yml:20]: '

**Comment by rimrul** [comment]
Should be good now.

**Comment by dscho** [comment]
Procedural note: since we need to rebuild cURL (after the dependency nghttp2 was built with a different DLL file name), I want to wait for https://dev.azure.com/git-for-windows/git/_build/results?buildId=108304&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a to finish, then kick off the two `sync` workflows to update `git-sdk-64`/`git-sdk-32`, and _then_ merge this PR so that the [next snapshot](https://wingit.blob.core.windows.net/files/index.html) will be built correctly. Just in case anybody was wondering why I do not merge this PR immediately.

**Pull request merged**